Clicker/ViewControllers/SettingsViewController.swift

```diff
@@ -0,0 +1,152 @@
+//
+//  SettingsViewController.swift
+//  Clicker
+//
+//  Created by eoin on 9/13/18.
+//  Copyright © 2018 CornellAppDev. All rights reserved.
+//
+
+import UIKit
+import IGListKit
+import GoogleSignIn
+
+class SettingsViewController: UIViewController {
+
+    // MARK: Views and VC's
+    var logOutButton: UIButton!
+    var collectionView: UICollectionView!
+    var lineView: UIView!
+    var adapter: ListAdapter!
+
+    // MARK: Data
+    var data: [SettingsDataModel]!
+    
+    // MARK: Layout constants
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        loadData()
+        setupNavBar()
+        setupViews()
+        setupConstraints()
+    }
+    
+    func loadData() {
+        let s1 = SettingsDataModel(state: .info, title: "Account", description: User.currentUser?.email)
+        let s2 = SettingsDataModel(state: .info, title: "About", description:
+            "Pollo is made by Cornell AppDev, an engineering project team at Cornell University.")
+        let s3 = SettingsDataModel(state: .link, title: "More Apps", description: "https://www.cornellappdev.com/")
+        let s4 = SettingsDataModel(state: .link, title: "Visit Our Website")
```

Comment:
make these strings class `let` variables. also can we name it to something like `settingsModel4` instead of `s4`, just for readibility